src/PKSim.CLI.Core/Services/QualificationRunner.cs

```diff
@@ -170,7 +173,7 @@ private IEnumerable<PlotMapping> retrieveChartDefinitionsForSimulation(Simulatio
       private Task<ObservedDataMapping[]> exportAllObservedData(PKSimProject project, QualifcationConfiguration configuration)
       {
          if (!project.AllObservedData.Any())
-            return Task.FromResult<ObservedDataMapping[]>(null);
```

Comment:
I was returning null. It should be empty array otherwise it makes life much harder in the parent code (qualificationrunner) when using unions.

THis files is a temp file that is an interface between the tool. We can assume the contact is that no array is null